pkg/util/provider/machinecontroller/machine_bootstrap_token.go

```diff
@@ -72,7 +72,7 @@ func (c *controller) addBootstrapTokenToUserData(ctx context.Context, machineNam
 		klog.V(4).Infof("replacing url encoded placeholder %s with %s in user-data!", urlEncodedPlaceholder, url.QueryEscape(token))
 		userDataS = strings.ReplaceAll(userDataS, urlEncodedPlaceholder, url.QueryEscape(token))
 	} else {
-		klog.V(4).Info("no bootstrap token placeholder found in user-data, nothing to replace!")
+		klog.Warningf("no bootstrap token placeholder found in user-data, nothing to replace!")
```

Comment:
Do we need to take action on this warning. 
Can we may be also include a small description of impact of this missing bootstrap token.

Reply:
We will still be creating machine in this case , as user might configure other ways of making bootstrap token available.
I have improved the message though , 8f6021a981cb977964ca97d6dc827f718c0314ff

Reply:
Thanks.
/lgtm